nixos/modules/services/desktops/gnome/gnome-keyring.nix

```diff
@@ -1,53 +1,48 @@
 # GNOME Keyring daemon.
 
-{ config, pkgs, lib, ... }:
-
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+let
+  cfg = config.services.gnome.gnome-keyring;
+in
 {
 
   meta = {
     maintainers = lib.teams.gnome.members;
   };
 
-  ###### interface
-
   options = {
-
     services.gnome.gnome-keyring = {
-
-      enable = lib.mkOption {
-        type = lib.types.bool;
-        default = false;
-        description = ''
-          Whether to enable GNOME Keyring daemon, a service designed to
-          take care of the user's security credentials,
-          such as user names and passwords.
-        '';
-      };
-
+      enable = lib.mkEnableOption ''
+        GNOME Keyring daemon, a service designed to
+        take care of the user's security credentials,
+        such as user names and passwords.
+      '';
     };
-
   };
 
+  config = lib.mkIf cfg.enable {
+    environment.systemPackages = with pkgs; [ gnome.gnome-keyring ];
 
-  ###### implementation
-
-  config = lib.mkIf config.services.gnome.gnome-keyring.enable {
-
-    environment.systemPackages = [ pkgs.gnome.gnome-keyring ];
+    services.dbus.packages = with pkgs; [
```

Comment:
`with` makes reference checking lazy so it can hide errors. I would avoid it here since it does not add much benefit here and actually makes the code slightly longer.